src/nifgen/metadata/functions_addon.py

```diff
@@ -0,0 +1,72 @@
+# These dictionaries are applied to the generated functions dictionary at build time
+# Any changes to the API should be made here. functions.py is code generated
+
+# By default all functions in functions.py will be generated as a public function
+# This will override that with private - add '_' to the beginning of the name, or
+# don't generate at all
+functions_codegen_method = {
```

Comment:
For a separate PR (you may want to add a TODO) there's still a good amount of functions to add here, I think.

Reply:
TODO added